src/pmsearch/pmsearch_index.sh

```
@@ -0,0 +1,235 @@
+
+ARGS=`pmgetopt --progname=$prog --config=$tmp/usage -- "$@"` || exit 1
+eval set -- "$ARGS"
+while [ $# -gt 0 ]
+do
+    case "$1"
+    in
+	-a)	ARCHIVE="$2"; shift ;;
+	-N)	SHOWME=true ;;
+	-o)	INDEX="$2"; shift ;;
+	-V)	VERBOSE=true ;;
+	-\?)	_usage; status=0; exit ;;
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)